setup.py

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env python
+# Copyright 2015: Mirantis Inc.
+# All Rights Reserved.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+
+from setuptools import setup
+from setuptools import find_packages
+from pip.req import parse_requirements
+
+setup(name='CloudFerry',
+      version=open('version.txt').read().strip(),
```

Comment:
I just open [PEP 0396 -- Module Version Numbers](https://www.python.org/dev/peps/pep-0396/#distutils2) and found that you can specify version in `version.txt` file. We can easily specify version via Jenkins job.
Also we don't use distutils2 that's why I `open().read()` from file directly.